tools/integration/test/serviceTest.js

```diff
@@ -0,0 +1,73 @@
+// (c) Copyright 2024, SAP SE and ClearlyDefined contributors. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+const { omit, isEqual } = require('lodash')
+const expect = require('chai').expect
```

Comment:
You could also be consistent with the other imports here, if you want 🙂

```suggestion
const { expect } = require('chai')
```